### PR TITLE
feat: [ListV2] Updated fee tooltip style

### PR DESCRIPTION
**Pull request by cbachmeier**
Updated the fee tooltip to the latest mocks

Screenshots:
Screen Shot 2023-02-14 at 14 15 30
Screen Shot 2023-02-14 at 14 15 45
Screen Shot 2023-02-14 at 14 17 13

### CODE REVIEW

File: src/nft/components/profile/list/RoyaltyTooltip.tsx [hunk 1]
```diff
@@ -1,27 +1,87 @@
...
   margin-bottom: 4px;
-  color: ${({ theme }) => theme.textPrimary};
+  justify-content: space-between;
 `
 
-const RoyaltyContainer = styled(ThemedText.BodySmall)`
-  margin-bottom: 8px;
+const RoyaltyContainer = styled(Column)`
+  gap: 12px;
+  padding 4px 0px;
```
Comment (JackShort) [inline src/nft/components/profile/list/RoyaltyTooltip.tsx:16]: `padding: 4px 0px;` - missing colon

File: src/nft/components/profile/list/RoyaltyTooltip.tsx [hunk 2]
```diff
@@ -1,27 +1,87 @@
...
+          </ThemedText.Caption>
+        </Row>
+        <FeePercent>{maxRoyalty}%</FeePercent>
       </FeeWrap>
+      <MaxFeeContainer>
+        <ThemedText.Caption lineHeight="16px">
+          <Trans>Max fees</Trans>
+        </ThemedText.Caption>
+        <ThemedText.Caption lineHeight="16px" color={fees ? 'textPrimary' : 'textSecondary'}>
+          {fees ? formatEth(fees) : '-'} ETH
```
Comment (JackShort) [inline src/nft/components/profile/list/RoyaltyTooltip.tsx:82]: the number formatter util will work for this I think

**Review by JackShort — APPROVED** [review]
small fixes

File: src/nft/components/profile/list/RoyaltyTooltip.tsx [hunk 3]
```diff
@@ -1,27 +1,87 @@
...
+  border-radius: 50%;
+`
+
+const FeePercent = styled(ThemedText.Caption)`
+  line-height: 16px;
+  color: ${({ theme }) => theme.textSecondary};
+  white-space: nowrap;
+`
+
+const MaxFeeContainer = styled(Row)`
```
Comment (JackShort) [inline src/nft/components/profile/list/RoyaltyTooltip.tsx:38]: maybe add `cursor: default` so you don't get the text cursor

Comment (cbachmeier) [reply src/nft/components/profile/list/RoyaltyTooltip.tsx:38]: the text cursor is the standard for all the elements in the grid so I think it's best to keep it for the fee as well to maintain consistency

File: src/nft/components/profile/list/RoyaltyTooltip.tsx (hunk 2 above)
Comment (cbachmeier) [reply src/nft/components/profile/list/RoyaltyTooltip.tsx:82]: the formatter unfortunately doesn't have the rounding to K or M for large numbers
ethNumberStandardFormatter
Screen Shot 2023-02-15 at 14 03 51
formatEth
Screen Shot 2023-02-15 at 14 04 05

**Pull request merged**